exercises/perfectShuffle.js

```diff
@@ -9,6 +9,22 @@
 
 function perfectShuffle(array) {
   // This is your job. :)
+  let firstHalfDeck = [];
+  let secondHalfDeck = [];
+  let halfDeck = array.length/2;
+  let shuffledDeck = [];
+  for (let i = 0; i < array.length; i++) {
```

Comment:
This works! You would also use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice